assets/src/application/ui/input/Button/Link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Tooltip } from '@infraUI/display';
+
+import './style.scss';
+
+interface LinkProps {
+	className?: string;
+	external?: boolean;
+	href?: string;
+	icon?: React.ReactNode;
+	title?: string;
+}
+
+const Link: React.FC<LinkProps> = ({ external, href, icon, title, ...props }) => {
```

Comment:
this was needed to take advantage of some icon button styles and to abstract away those anchor attr from `RegistrationsLink`